app/controllers/katello/api/v2/host_subscriptions_controller.rb

```diff
@@ -52,11 +52,11 @@ def auto_attach
       respond_for_index(:collection => index_response, :template => "index")
     end
 
-    api :GET, "/hosts/:host_id/subscriptions/events", N_("List subscription events for the host")
+    api :GET, "/hosts/:host_id/subscriptions/events", N_("List subscription events for the host"), :deprecated => true
     param :host_id, Integer, :desc => N_("Id of the host"), :required => true
     def events
-      collection = full_result_response(@host.subscription_facet.candlepin_consumer.events.map { |e| OpenStruct.new(e) })
-      respond_for_index :collection => collection
+      ::Foreman::Deprecation.api_deprecation_warning("Events API is being deprecated and will be removed in Katello 3.16.")
+      respond_for_index :collection => {results: []}
```

Comment:
Ugh, sorry @sseelam2 .  I noticed that with this, the paging and other result metadata in the response doesn't get populated. I think we need to use the `full_result_reponse` method like before, but with an empty collection. Try it out!

Reply:
ie collection => full_result_response([]) should do the trick

Reply:
@jturel this is ready to retest.